beetsplug/mpdstats.py

```diff
@@ -50,8 +50,9 @@ class MPDClientWrapper(object):
     def __init__(self, log):
         self._log = log
 
-        self.music_directory = (
-            mpd_config['music_directory'].as_str())
+        self.music_directory = (mpd_config['music_directory'].as_str())
+
+        self.strip_path = (mpd_config['strip_path'].as_str())
```

Comment:
```suggestion
        self.music_directory = mpd_config['music_directory'].as_str()
        self.strip_path = mpd_config['strip_path'].as_str()
```
No need for parentheses now (or the blank line).

Reply:
Fixed in last commit :)